test/flux_calculator_tests.jl

```diff
@@ -36,9 +36,9 @@ Interfacer.get_field(sim::TestAtmos, ::Val{:thermo_state_int}) =
     TD.PhaseEquil_ρTq.(get_thermo_params(sim), sim.integrator.ρ, sim.integrator.T, sim.integrator.q)
 
 function FieldExchanger.update_sim!(sim::TestAtmos, fields, _)
-    (; F_turb_ρτxz, F_turb_energy, F_turb_moisture) = fields
+    (; F_turb_ρτxz, F_lhf, F_shf, F_turb_moisture) = fields
     ρ_int = sim.integrator.ρ
-    @. sim.integrator.p.energy_bc = -(F_turb_energy)
+    @. sim.integrator.p.energy_bc = -(F_lhf + F_shf)
```

Comment:
out of curiosity, why the minus sign?

Reply:
I'm not sure! I thought it might be a sign convention, but when we update the fluxes for the real atmosphere, we don't negate them (e.g. [here](https://github.com/CliMA/ClimaCoupler.jl/blob/e991496e49ba355c212ed4cf8291801f7bd9fc44/experiments/ClimaEarth/components/atmosphere/climaatmos.jl#L363-L365)). This test is not very meaningful - I removed all the minus signs here and it still passes.

I'll keep it without the minus signs, since that seems to match our simulation code more closely